src/mscorlib/model.xml

```diff
@@ -6202,15 +6202,15 @@
       <Member Name="StringToHGlobalUni(System.String)" />
       <Member Name="StructureToPtr&lt;T&gt;(T,System.IntPtr,System.Boolean)" />
       <Member Name="UnsafeAddrOfPinnedArrayElement&lt;T&gt;(T[],System.Int32)" />
-      <Member Name="WriteByte(System.Object,System.Int32,System.Byte)" />
+      <Member Name="WriteByte(System.Object,System.Int32,System.Byte)" Condition="FEATURE_LEGACYSURFACE"/>
       <Member Name="WriteInt16(System.IntPtr,System.Char)" />
       <Member Name="WriteInt16(System.IntPtr,System.Int32,System.Char)" />
-      <Member Name="WriteInt16(System.Object,System.Int32,System.Char)" />
-      <Member Name="WriteInt16(System.Object,System.Int32,System.Int16)" />
-      <Member Name="WriteInt32(System.Object,System.Int32,System.Int32)" />
-      <Member Name="WriteInt64(System.Object,System.Int32,System.Int64)" />
+      <Member Name="WriteInt16(System.Object,System.Int32,System.Char)" Condition="FEATURE_LEGACYSURFACE"/>
+      <Member Name="WriteInt16(System.Object,System.Int32,System.Int16)" Condition="FEATURE_LEGACYSURFACE"/>
+      <Member Name="WriteInt32(System.Object,System.Int32,System.Int32)" Condition="FEATURE_LEGACYSURFACE"/>
+      <Member Name="WriteInt64(System.Object,System.Int32,System.Int64)" Condition="FEATURE_LEGACYSURFACE"/>
```

Comment:
Nit - could you please add space between " and /> to make the formatting consistent?